wg-ecosystem/meeting-notes/2020-02-06.md

```diff
@@ -0,0 +1,32 @@
+# Ecosystem WG meeting 2020-02-06
+
+## Attendees
+
+* @HashimotoYT
+* @ckerr
+* @erickzhao
+* @felixrieseberg
+* @malept
+* @marshallofsound
+
+## Agenda
+
+* Docs feedback: Appzi/LoopInput
+  * Table discussion until we communicate with Appzi
+* Reverting GitHub Action Xvfb PR (https://github.com/electron/electron/pull/21996)
+  * Preventing future incidents: use `pending-vote` label?
+* Key Result status updates
+  * Increase adoption of Electron-provided tooling (@ckerr)
+    * Call for ideas.
+      * @felixrieseberg will followup in `#wg-ecosystem` with some ideas (e.g. Fiddle doesn't work as well without a good internet connection)
+      * @malept mentions we could promote our own tooling better in our documentation
+      * @ckerr mentions example of that: build-tools isn't promoted in the "Building on Platform X" docs
+  * Increase views on /docs (@felix)
+* [Open pull requests for the WG to review](https://github.com/pulls?q=is%3Apr+team-review-requested%3Aelectron%2Fwg-ecosystem+archived%3Afalse+is%3Aopen+)
+    * https://github.com/electron/electron/pull/22034
+* [Open pull requests mentioning the WG in a comment](https://github.com/pulls?q=is%3Apr+team%3Aelectron%2Fwg-ecosystem+archived%3Afalse+is%3Aopen)
+
+## Action Items
```

Comment:
Just remembered this -- didn't we also assign @ckerr to update the Development docs to include build-tools?

Reply:
Well, he assigned himself 😆 I can add that.